sys/Makefile.include

```diff
@@ -31,6 +31,13 @@ ifneq (,$(filter oneway_malloc,$(USEMODULE)))
   USEMODULE_INCLUDES += $(RIOTBASE)/sys/oneway-malloc/include
 endif
 
+ifneq (,$(filter metadata,$(USEMODULE)))
+  # Overwrite the interface version
```

Comment:
what interface is this referring to? API?

Reply:
None right now, the plan it to have the RDM merged then we can specify it adheres to the rules.  Similar to something like it adheres to a [JSON schema draft 7](https://json-schema.org/draft-07/json-schema-release-notes.html).

Reply:
What RDM? There are 3 under review right now?

Reply:
> None right now, the plan it to have the RDM merged then we can specify it adheres to the rules. Similar to something like it adheres to a [JSON schema draft 7](https://json-schema.org/draft-07/json-schema-release-notes.html).

Though I like the simplicity of this implementation, I wonder

a) Why pin-point to JSON, when we also support e.g. CBOR?
b) Why use `printf` and not `fmt` *iff* we want to stick to JSON.

Reply:
The RDM I am referring to is #10624